roles/etcd/defaults/main.yaml

```diff
@@ -16,7 +16,11 @@ etcd_image: "{{ etcd_image_dict[openshift_deployment_type | default('origin')] }
 # etcd run on a host => use etcdctl command directly
 # etcd run as a docker container => use docker exec
 # etcd run as a runc container => use runc exec
-r_etcd_common_etcdctl_command: "{{ 'etcdctl' if r_etcd_common_etcd_runtime == 'host' | bool else 'docker exec etcd_container etcdctl' if r_etcd_common_etcd_runtime == 'docker' else 'runc exec etcd etcdctl' }}"
+etcdctl_dict:
+  host: 'etcdctl'
+  docker: 'docker exec etcd_container etcdctl'
+  runc: 'runc exec etcd etcdctl'
+r_etcd_common_etcdctl_command: "{{ etcdctl_dict[r_etcd_common_etcd_runtime | default('runc')] }}"
```

Comment:
Why default to runc here rather than host which I imagine is more common?

Reply:
Right, the code used to be a bit confusing - `r_etcd_common_etcd_runtime` would default to host if no other matches found, but `r_etcd_common_etcdctl_command` would default to `runc` if other checks failed.

Updated the code to default to `host`